src/components/sidebar/components/index.ts

```diff
@@ -0,0 +1,5 @@
+import SidebarHorizontalRule from './sidebar-horizontal-rule'
+import SidebarSectionHeading from './sidebar-section-heading'
+import SidebarTitleHeading from './sidebar-title-heading'
+
+export { SidebarHorizontalRule, SidebarSectionHeading, SidebarTitleHeading }
```

Comment:
Love me a barrel export file ❤️

Reply:
Me too! ✨  Wish we could use one for our main `src/components` but I think it had negative impacts on the build time. 😢

Reply:
+1 this is cool and also kinda new to me and I like it 💖